src/app/admin/modals/user-notification-settings-modal/user-notification-settings.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { MatDialogRef, MatDialog } from '@angular/material/dialog';
+import { notificationComponent } from './notification.component';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserNotificationSettingsModalService {
+  constructor(public dialog: MatDialog) {}
+
+  public show(task: any) {
+    let dialogRef: MatDialogRef<notificationComponent, any>;
+    dialogRef = this.dialog.open(notificationComponent, {
+      panelClass: 'custom-dialog-class',
+    });
+  }
+}
```

Comment:
Can we add a newline at the end of the file?

Reply:
Thanks for your suggestions

Reply:
empty line added